h2o-assembly/build.gradle

```diff
@@ -10,6 +10,7 @@ dependencies {
   if (project.hasProperty("doIncludeOrc") && project.doIncludeOrc == "true") {
     compile project(":h2o-orc-parser")
   }
+  compile project(":h2o-parquet-parser")
```

Comment:
Agreed, fortunately parquet has very reasonable set of dependencies. 

These are the compile dependencies of project h2o-parquet-parser:

\--- project :h2o-parquet-parser
     +--- project :h2o-core (*)
     +--- org.apache.parquet:parquet-hadoop:1.7.0
     |    +--- org.apache.parquet:parquet-column:1.7.0
     |    |    +--- org.apache.parquet:parquet-common:1.7.0
     |    |    +--- org.apache.parquet:parquet-encoding:1.7.0
     |    |    |    +--- org.apache.parquet:parquet-common:1.7.0
     |    |    |    +--- org.apache.parquet:parquet-generator:1.7.0
     |    |    |    |    \--- org.apache.parquet:parquet-common:1.7.0
     |    |    |    \--- commons-codec:commons-codec:1.5 -> 1.6
     |    |    \--- commons-codec:commons-codec:1.5 -> 1.6
     |    +--- org.apache.parquet:parquet-format:2.3.0-incubating
     |    +--- org.apache.parquet:parquet-jackson:1.7.0
     |    +--- org.codehaus.jackson:jackson-mapper-asl:1.9.11 -> 1.9.13 (*)
     |    +--- org.codehaus.jackson:jackson-core-asl:1.9.11 -> 1.9.13
     |    \--- org.xerial.snappy:snappy-java:1.1.1.6
     +--- org.apache.parquet:parquet-avro:1.7.0
     |    +--- org.apache.parquet:parquet-column:1.7.0 (*)
     |    +--- org.apache.parquet:parquet-hadoop:1.7.0 (*)
     |    +--- org.apache.parquet:parquet-format:2.3.0-incubating
     |    \--- org.apache.avro:avro:1.7.6 -> 1.8.0 (*)
     \--- org.apache.hadoop:hadoop-common:2.6.0-cdh5.4.0 -> 2.7.1.2.4.0.0-169 (*)